spec/features/create_object_no_files_spec.rb

```diff
@@ -35,6 +35,7 @@
     reload_page_until_timeout!(text: 'v1 Registered', with_reindex: true)
 
     # add accessionWF
+    expect(page).to have_text('Successfully updated index for') # ensure we see this first
```

Comment:
Isn't seeing "v1 Registered" on line 35 enough to indicate we are on the show page?  Why do we need to also look for this?

Reply:
because it's a race condition:  if the wf is added BEFORE the "Successfully updated index" message, then we don't see the "added AccessionWF" message ever.  I was hitting this over and over yesterday.

Reply:
I'm having trouble understanding the race condition here.  Can you explain it to me?  Line 35 should loop over reindexing N times until `v1 Registered` is on the page.  Only then will it add the workflow, so, how is it racing?

Reply:
If v1 is accessioning, but the 'Successfully updated index for' message doesn't appear before we add the workflow, then the workflow is added, but the message 'Added accessionWF' may never display because it is stomped over by 'Successfully updated index' message.   I saw this happen a couple times:   the test hung waiting for 'Added accessionWF' message but I could see 'Successfully updated index' as the message and the accessioning WF had started.

Reply:
Why is "Successfully updated index for" appearing?  That shouldn't happen once `v1 Registered` appears.  This sounds like a bug in `reload_page_until_timeout!`

Reply:
Maybe the check for "Successfully updated index" should go here: https://github.com/sul-dlss/infrastructure-integration-test/blob/216d6d675c1ec4bed5ae384e93f924c407944057/spec/support/page_helpers.rb#L14

Reply:
good idea - i'll do that.